docs/user-guide/global-configurations/cluster-and-environments.md

```diff
@@ -1,144 +1,311 @@
 # Clusters and Environments
 
-You can add your existing Kubernetes clusters and environments on the `Clusters and Environments` section. You must have a [super admin](./authorization/user-access.md#assign-super-admin-permissions) access to add a cluster.
+## Introduction
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/cluster-and-environments/cluster-and-environments.png)
+Devtron allows you to connect and manage your existing Kubernetes clusters by adding them to its platform. Once a cluster is added, you can create different environments within it, making it possible to deploy your applications.
 
-## Add Cluster:
+Go to **Global Configurations** → **Clusters & Environments** → **Add Cluster** (button)
 
-To add cluster, go to the `Clusters & Environments` section of `Global Configurations`. Click **Add cluster**.
+![Figure 1: Adding a Cluster](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/cluster-and-environments/add-clusters.jpg)
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/cluster-and-environments/add-clusters.png)
+You can add any of the following cluster types:
+* [Kubernetes Cluster](#add-kubernetes-cluster) - This is the recommended option for adding a cluster.
```

Comment:
Why is this a recommended option? It should be like if you have access choose this, if not choose that